rerun_js/web-viewer/build-wasm.mjs

```diff
@@ -31,7 +31,8 @@ function buildWebViewer(mode) {
       "cargo run -p re_dev_tools -- build-web-viewer",
       modeFlags,
       "--target no-modules-base",
-      "--features grpc",
+      "--no-default-features",
+      "--features grpc,map_view", // no `analytics`
```

Comment:
why not?

Reply:
this is for the NPM package, we don't want users' applications which embed the web viewer this way to send analytics to us. they will only do that if they embed the web viewer as an iframe